webapp/src/views/DomainList.vue

```diff
@@ -23,6 +23,7 @@ export default {
           destroy: 'Domain Deletion',
         },
         texts: {
+          banner: () => ('You can create and delete domains here. We will soon extend our GUI to offer DNS record management. In the meantime, please use the API to manage records.'),
```

Comment:
add link to the docs?

Reply:
I don't think you can add HTML here

Reply:
I've seen it elsewhere, maybe it's worth a try

Reply:
worked by adding a `span`.

Reply:
nice :+1: